Lib/email/_policybase.py

```diff
@@ -307,8 +307,8 @@ def header_source_parse(self, sourcelines):
 
         """
         name, value = sourcelines[0].split(':', 1)
-        value = value.lstrip(' \t') + ''.join(sourcelines[1:])
-        return (name, value.rstrip('\r\n'))
+        value = value + ''.join(sourcelines[1:])
+        return (name, value.lstrip(' \r\n\t').rstrip('\r\n'))
```

Comment:
How can ``sourcelines[0]`` have either a carriage return or line feed, if it is the first element of a list of lines? 
```suggestion
        value = value.lstrip(' \r\n\t') + ''.join(sourcelines[1:])
        return (name, value.rstrip('\r\n'))
```

Reply:
I moved lstrip at the end because the whitespace could also appear in subsequent newlines that follows the initial line. (This is what happens in the testcase added, with leading \r\n\t)